docs/tx_rollup.md

```diff
@@ -0,0 +1,29 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 Oxhead Alpha
+   - SPDX-License-Identifier: LicenseRef-MIT-OA
+   -->
+
```

Comment:
IMO it would be nice to mention this document in the README, maybe close by to the link to the baking doc